### ISSUE TITLE
Subscription Management Not Checking For 'Unsubscribed' Condition

### ISSUE THREAD

**Issue by I-am-Sherlocked** [TAXIIProject/django-taxii-services#42]
In message_handlers.SubscriptionRequest11Handler , there is no condition to check for the current status in "pause" or "resume" methods. That seems wrong to me as I don't believe the expected behavior is for the subscription to be paused or resumed (i.e. back to ACTIVE state) after it was unsubscribed, which is what happens right now.

**Comment by MarkDavidson** [comment]
@I-am-Sherlocked,

The Subscription Management handlers are definitely not well implemented or well tested (you may notice all the TODOs in the code). Thank you for reporting this.

**Comment by I-am-Sherlocked** [comment]
I know, I know. But there wasn't any TODOs for this specific issue, so I figured let's document it :)

**Comment by MarkDavidson** [comment]
@I-am-Sherlocked Yes, please do. Documented things get done before undocumented things!
